src/Forms/Container.php

```diff
@@ -255,6 +255,9 @@ public function getForm(bool $throw = true): ?Form
 	 */
 	public function addText(string $name, $label = null, int $cols = null, int $maxLength = null): Controls\TextInput
 	{
+		if (class_exists(Debugger::class) && in_array($name, Debugger::getBlueScreen()->keysToHide, true)) {
+			trigger_error('Text input "' . $name . '" should be type of password.', E_USER_WARNING);
+		}
```

Comment:
That's a hidden dependency which may be not easy to fix.

Reply:
Also I am sure even beginner can easily notice that typed password is not hidden.